mobify/progressive-web-scaffold
# Use SDK router

The SDK will now wrap the entire react-router API and export the useful parts from there. This changes the scaffold to use it and removes the direct dependencies on `react-router` and `react-router-scroll`.

 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/93 (merged), https://github.com/mobify/progressive-web-sdk/pull/95

## Changes
- Convert router components to the SDK versions
- Remove now-unneeded deps

## How to test-drive this PR
- Connect to a version of the SDK with the above fixes.
- Build and preview Merlin's Potions
- Ensure that the page renders and the links work.
